interfaces/builtin/unity7.go

```diff
@@ -145,6 +145,12 @@ dbus (receive,send)
     member=Changed
     peer=(label=unconfined),
 
+dbus (send)
+    bus=session
+    interface=com.canonical.UrlLauncher.XdgOpen
+    peer=(label=unconfined),
```

Comment:
Can you comment on what this does, the XdgOpen API, etc? This rule is awfully open ended and seems like it would allow an application to launch arbitrary other applications, perhaps with arguments.